app/presenters/instance_presenter.rb

```diff
@@ -50,6 +50,13 @@ def rules
     Rule.ordered
   end
 
+  def report_categories
+    # We only return the `violation` category if we have rules to violate:
+    Report.categories.filter_map do |category, _value|
```

Comment:
Could see this being a class method on `Report` ... feels vaguely similar to `UserRole.providing_styles` (used in custom css controller).

Reply:
We realised separately that this probably needs to include the localised text, but we don't have any existing APIs that localise responses iirc. I think that was the blocker here